android/gradle.properties

```diff
@@ -26,3 +26,8 @@ android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.93.0
+
+MYAPP_UPLOAD_STORE_FILE=my-upload-key.keystore
+MYAPP_UPLOAD_KEY_ALIAS=my-key-alias
+MYAPP_UPLOAD_STORE_PASSWORD=coditas@123
```

Comment:
This file contains sensitive information such as passwords, so it'll be nice to keep this file out of the version control system. 
Add this file in .gitignore

Reply:
created .gitignore file and untracked gradle.propertied